windows/utils/win_collect_guest_file.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023-2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   Collect VMWare tools logs to local test case log directory
```

Comment:
This task file name is `win_collect_guets_file.yml` but the description said it collects VMware Tools logs. They are not consistent.